src/Propel/Generator/Behavior/Versionable/VersionableBehaviorObjectBuilderModifier.php

```diff
@@ -197,6 +278,11 @@ public function getVersion()
 ";
     }
 
+    /**
+     * @param string $script
```

Comment:
Might be worth increasing the visibility that this is by-reference. `&$script`, what do you think? Does the tool support it?

Reply:
This is not valid afaik. The variable is still only this. You can check by trying it, afaik phpstan or psalm will call this out as not valid docblock.

Reply:
:+1: